spec/lib/authorities/linked_data/config_spec.rb

```diff
@@ -405,8 +404,7 @@
   describe '#prefixes' do
     let(:expected_results) do
       {
-        schema: "http://www.w3.org/2000/01/rdf-schema#",
-        skos: "http://www.w3.org/2004/02/skos/core#"
+        dcterms: "http://purl.org/dc/terms/"
```

Comment:
rdfs (previously prefixed as schema) and skos are predefined in the ldpath gem and do not need to be defined here.  dcterms is not predefined and needs to be defined here to be used in the config.